src/test/java/org/broadinstitute/hellbender/tools/walkers/haplotypecaller/graphs/ChainPrunerUnitTest.java

```diff
@@ -156,8 +156,8 @@ public void testAdaptivePruning(final int kmerSize, final byte[] ref, final byte
         pruner.pruneLowWeightChains(graph);
 
         final SmithWatermanAligner aligner = SmithWatermanJavaAligner.getInstance();
-        graph.recoverDanglingTails(1, 3, aligner);
-        graph.recoverDanglingHeads(1, 3, aligner);
+        graph.recoverDanglingTails(1, 3, false, aligner);
+        graph.recoverDanglingHeads(1, 3, false, aligner);
```

Comment:
Would it be difficult to add a unit test where `recoverAll` is `true`?

Reply:
I think that can be done.

Reply:
done.